.pre-commit-config.yaml

```diff
@@ -1,10 +1,26 @@
+exclude: "(migrations|static/node_modules|static/assets|static/vendor)"
 repos:
--   repo: https://github.com/psf/black
-    rev: 22.1.0
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
     hooks:
-    - id: black
-      language_version: python3.7
--   repo: https://github.com/pycqa/flake8
-    rev: 4.0.1
+      - id: check-yaml
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
     hooks:
-    -   id: flake8
+      - id: black
+        language_version: "3.10"
+  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
should flake8 be replaced by pylint?

Reply:
I've seen cases where flake8 catches stuff that pyling doesn't and viceversa. I would keep them both for some time and see how it goes.